packages/frontend/components/ui/checkbox.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cls from 'classnames';
+
+function Checkbox({
+  name,
+  checked,
+  type,
+  disabled,
+  className,
+  value,
+  label,
+  required,
+  onChange,
+  ...otherProps
+}) {
+  function changeValue(targetValue) {
+    if (onChange) {
+      onChange(targetValue);
+    }
+  }
+
+  let customStyleName = '';
+  if (checked && type === 'success') {
+    customStyleName = 'success-check-box';
+  } else if (checked && type === 'danger') {
+    customStyleName = 'danger-check-box';
+  }
+
+  return (
+    <div className="flex items-center">
+      <input
+        id={name}
+        name={name}
+        type="checkbox"
+        checked={checked}
+        className={cls(`default-check-box ${customStyleName}`, className)}
```

Comment:
can you write this like `cls('default-check-box', customStyleName, className)` to catch our standarts please?